py_opt_cli/__init__.py

```diff
@@ -409,7 +409,7 @@ class Change(OptimizelyDocument):
     dependencies = attr.ib()
     id = attr.ib()
     type = attr.ib()
-    async = attr.ib(default=None)
+    # async = attr.ib(default=None)
```

Comment:
@cpennington now `async` is a keyword, should this be renamed?

Reply:
Yeah... I'm not sure if `attrs` can handle renames. The difficulty is that that is a value coming in from the json from optimizely.

Reply:
I was hoping that `attrs` supported a `name` parameter, but no such luck: https://github.com/python-attrs/attrs/issues/381

I also took a swing at using the `these` parameter to `attr.s`, but no such luck -- it creates a string template that it then evals, rather than using setattr:
```
>>> @attr.s(these={'b': attr.ib(), 'async': attr.ib()})
... class Bar:
...     pass
... 
Traceback (most recent call last):
[...]
  File "/home/timmc/adhoc/venv-3.8/lib/python3.8/site-packages/attr/_make.py", line 299, in _make_attr_tuple_class
    eval(compile("\n".join(attr_class_template), "", "exec"), globs)
  File "", line 4
    async = _attrs_property(_attrs_itemgetter(1))
          ^
SyntaxError: invalid syntax
```

One option would be:

- Change those `subdocuments(Change)` calls so that certain fields are remapped (`"async"` to `"is_async"` or whatever) on copies of the input dicts, maybe by consulting an optional class method; and
- Change `OptimizelyDocument`'s reader and writer methods to perform this remapping as well.